charts/cluster-autoscaler/README.md

```diff
@@ -316,6 +316,7 @@ Though enough for the majority of installations, the default PodSecurityPolicy _
 | deployment.annotations | object | `{}` | Annotations to add to the Deployment object. |
 | dnsPolicy | string | `"ClusterFirst"` | Defaults to `ClusterFirst`. Valid values are: `ClusterFirstWithHostNet`, `ClusterFirst`, `Default` or `None`. If autoscaler does not depend on cluster DNS, recommended to set this to `Default`. |
 | envFromConfigMap | string | `""` | ConfigMap name to use as envFrom. |
+| envFromConfigMapOptional | boolean | `true` | Whether or not the ConfigMap in envFrom is optional. |
```

Comment:
@jebbens This takes its description from the comment about the variable in values.yaml via `helm-docs` (see our [pre-commit config](https://github.com/kubernetes/autoscaler/tree/master/charts#pre-commit-hooks)), can you update the values.yaml to reflect this explanation? This is the reason the PR check is currently failing.

Reply:
Pushed a new commit - I think this is what you are looking for